src/snowflake/snowpark/_internal/data_source/utils.py

```diff
@@ -53,6 +53,16 @@ class DBMS_TYPE(Enum):
     "sqlite3": SqliteDriver,
 }
 
+UDTF_PACKAGE_MAP = {
+    DBMS_TYPE.ORACLE_DB: ["oracledb", "snowflake-snowpark-python"],
+    DBMS_TYPE.SQLITE_DB: ["snowflake-snowpark-python"],
+    DBMS_TYPE.SQL_SERVER_DB: [
+        "pyodbc>=4.0.26",
```

Comment:
why we want `> 4.0.26`?

Reply:
cause get_output_converter is only supported in >=4.0.26

Reply:
I assume you have tested the version available in our anaconda is already >= 4.0.26?